subsystems/sidecar/index.js

```diff
@@ -693,6 +661,7 @@ class Sidecar extends ReadyResource {
     const id = client.userData?.id || `${client.id}@${startId}`
     const app = client.userData = client.userData?.id ? client.userData : new this.App({ id, startId, session })
     const state = new State({ id, env, link, dir, cwd, flags, args, cmdArgs, run: true })
+    const trustedFlag = cmdArgs.some((arg) => arg === '--trusted')
```

Comment:
not needed - `flags.trusted`

Reply:
updated, I was setting the flag in the wrong place